### DISCUSSION TITLE
Slower speeds then Ookla and other Speedtest

### DISCUSSION THREAD

**Discussion by Flx1985** [alexjustesen/speedtest-tracker#632] [Q&A] [answered]
Hello,
i have a Problem with the Download Speed - i get round about 200Mbps - when i run the Test with ookla (Desktop App or Browser), i get round about 960Mbps - in the Settings i choose the same Server that ookla used. My Upload Speed ist correct. I run the Docker in Win10 - maybe some Config errors. I hope that someone can Help me.

Cheers Felix

**Comment by alexjustesen** [comment, marked as answer]
So Windows 10 and Docker have some networking bottlenecks. I strongly suggest not running it there. Unfortunately nothing I can fix.
